salt/modules/win_lgpo.py

```diff
@@ -5047,7 +5047,10 @@ def get(policy_class=None, return_full_policy_names=True,
                     class_vals[policy_name] = __salt__['reg.read_value'](_pol['Registry']['Hive'],
                                                                          _pol['Registry']['Path'],
                                                                          _pol['Registry']['Value'])['vdata']
-                    log.debug('Value {0} found for reg policy {1}'.format(class_vals[policy_name], policy_name))
+                    log.debug(
+                        'Value {0} found for reg policy {1}'.format(
+                            salt.utils.to_unicode(class_vals[policy_name]),
```

Comment:
If this is just binary data, doing a `to_unicode` will fail. What content does this value contain?

```
>>> salt.utils.stringutils.to_unicode('\xaf')
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "salt/utils/stringutils.py", line 109, in to_unicode
  File "/usr/lib/python2.7/encodings/utf_8.py", line 16, in decode
    return codecs.utf_8_decode(input, errors, True)
UnicodeDecodeError: 'utf8' codec can't decode byte 0xaf in position 0: invalid start byte
```

Reply:
It looks like it's string data. `reg.read_value` will return unicode or str.

Reply:
Yeah, but on Python 2, binary blobs are represented in `str` types. The fact that it's `unicode` or `str` doesn't mean that it won't contain binary data.

Reply:
If it's binary data, the `to_unicode` will fail, as I noted in my earlier comment. To prevent a decode error, we can just use `repr()`:

```python
log.debug(
    'Value %s found for reg policy %s',
    repr(class_vals[policy_name]), policy_name
)
```

Reply:
And actually I think you can use `%r` and not have to use `repr()`

Reply:
Yep, that's true... So, this is more concise:

```python
log.debug(
    'Value %r found for reg policy %s',
    class_vals[policy_name], policy_name
)
```